charts/kubernetes-external-secrets/templates/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "kubernetes-external-secrets.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "kubernetes-external-secrets.name" . }}
+    helm.sh/chart: {{ include "kubernetes-external-secrets.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "kubernetes-external-secrets.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "kubernetes-external-secrets.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+      annotations:
+{{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+      serviceAccountName: {{ template "kubernetes-external-secrets.fullname" . }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          env:
+            - name: AWS_REGION
```

Comment:
After https://github.com/godaddy/kubernetes-external-secrets/pull/51, we should remove `AWS_REGION` in favor of `AWS_DEFAULT_REGION`.
We should also add `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, for people that runs the daemon in a non-AWS environment and need to explicitly set AWS credentials (since IAM roles for ec2 and eks are not available).

Reply:
Also, let's be more flexible here and simply use `{{ toYaml .Values.env | indent 12 }}` instead.

Reply:
Yeah, fair point. This PR was a minimal, helm chart only addition without changes to the application. I just exposed a mechanism to set the env variables already been looked for in `config/environment.js`.

`nindent` is cool, didnt know about that addition 👍

Reply:
Going to mark this as resolved - changes to the application to look for additional AWS_* env vars can be addressed in a separate PR.